syncopy/specest/compRoutines.py

```diff
@@ -922,22 +936,87 @@ def fooofspy_cF(trl_dat, foi=None, timeAxis=0,
     --------
     syncopy.freqanalysis : parent metafunction
     """
+    if timeAxis != 0:
+        raise SPYValueError("timeaxis of input spectral data to be 0. Non-standard axes not supported with FOOOF.", actual=timeAxis)
+
     outShape = trl_dat.shape
     # For initialization of computational routine,
     # just return output shape and dtype
     if noCompute:
         return outShape, spectralDTypes['pow']
 
     # Call actual fooof method
-    res, _ = fooofspy(trl_dat[0, 0, :, :], in_freqs=fooof_settings['in_freqs'], freq_range=fooof_settings['freq_range'], out_type=output_fmt,
+    res, metadata = fooofspy(trl_dat[0, 0, :, :], in_freqs=fooof_settings['in_freqs'], freq_range=fooof_settings['freq_range'], out_type=output_fmt,
                       fooof_opt=method_kwargs)
 
-    # TODO (later): get the 'details' from the unused _ return
-    #  value and pass them on. This cannot be done right now due
-    #  to lack of support for several return values, see #140.
+    if 'settings_used' in metadata:
+        del metadata['settings_used']  # We like to keep this in the return value of the
```

Comment:
yes, they get added to the log in `freqanalysis()` already. But they cannot go into the hdf5. And I want to keep the backend function as-is, because they are very handy for the backend tests.